splice_machine/src/main/java/com/splicemachine/derby/impl/store/access/btree/IndexConglomerate.java

```diff
@@ -151,16 +155,25 @@ protected void create(boolean isExternal,
         }
         try{
 //            ((SpliceTransaction)rawtran).elevate(Bytes.toBytes(Long.toString(containerId)));
+
+            TxnView txnView = null;
+            if (rawtran instanceof SpliceTransaction) {
+                txnView = ((SpliceTransaction) rawtran).getTxnInformation();
+            }
+            else if (rawtran instanceof SpliceTransactionView) {
+                txnView = ((SpliceTransactionView) rawtran).getTxnInformation();
+            }
```

Comment:
TxnView does not seem to be used in the following code. Looks like line 169 covers both cases here.

Reply:
This has been deleted